services/app-api/forms/sar.json

```diff
@@ -1151,13 +1301,14 @@
             "isRequired": true,
             "verbiage": {
               "intro": {
-                "section": "Expenditures",
+                "section": "State- or Territory-Specific Initiatives",
```

Comment:
We going back to `State-` here or should it just be `State` (without hyphen)?

Reply:
thanks for the catch!